policies/tagging/policy.json

```diff
@@ -34,24 +34,14 @@
           "defaultValue": [
             "CFT",
             "crime",
-            "cross-cutting"
+            "Cross-Cutting"
```

Comment:
should this be all lowercase?

Reply:
The value is  cap for first letter in the [common module](https://github.com/hmcts/terraform-module-common-tags/blob/c4c3d2064c30f3bd1aeffa9cf50da315d07132fa/team-config.yml#L14) so updated here as well, my assumption is its case sensitive